front/app/containers/MainHeader/index.tsx

```diff
@@ -270,25 +104,13 @@ const MainHeader = ({ intl: { formatMessage } }: WrappedComponentProps) => {
     secondUrlSegment === 'initiatives' &&
     lastUrlSegment !== 'new';
   const isAdminPage = isPage('admin', location.pathname);
-  const isEmailSettingsPage = isPage('email-settings', location.pathname);
   const isProjectPage = !!(
     urlSegments.length === 3 &&
     urlSegments[0] === locale &&
     urlSegments[1] === 'projects'
   );
-  const isDesktopUser = isDesktop(windowSize.windowWidth);
-
-  const trackSignUpLinkClick = () => {
-    trackEventByName(tracks.clickSignUpLink.name);
-  };
 
-  const signIn = () => {
-    triggerAuthenticationFlow({ flow: 'signin' });
-  };
-
-  const signUp = () => {
-    triggerAuthenticationFlow({ flow: 'signup' });
-  };
+  const isDesktopUser = isDesktop(windowSize.windowWidth);
```

Comment:
```suggestion
const isSmallerThanDesktop = useBreakpoint('desktop');
 const isDesktopUser = !isSmallerThanDesktop;
```
We're trying to factor out `useWindowSize`.